lean/src/test.lean

```diff
@@ -275,11 +275,24 @@ begin
   sorry
 end
 
+lemma dvd_of_dvd_add_mul_left (a b n : ℕ) : a ∣ b + a * n → a ∣ b :=
+begin
+  refine (nat.dvd_add_left _).mp,
+  exact dvd_mul_right a n,
+end
+
 theorem induction_12dvd4expnp1p20
   (n : ℕ) :
   12 ∣ 4^(n+1) + 20 :=
 begin
-  sorry
+  induction n with k IH,
+  { dec_trivial },
+  {
+    rw pow_succ,
+    -- If we add 60 to RHS, then we can factor the 4 to use IH
+    apply dvd_of_dvd_add_mul_left 12 (4 * 4 ^ k.succ + 20) 5,
```

Comment:
Any chance you could fold the lemma into the proof?

Reply:
Yup, It's done!